cdap-docs/_common/_source/index.rst

```diff
@@ -68,17 +68,17 @@ on the installation, monitoring and diagnosing fully distributed CDAP in a Hadoo
   - **Advanced Topics:** Best practices for CDAP development, adding a custom logback
 
 
-.. |included-applications| replace:: **Included Applications:**
-.. _included-applications: included-applications/index.html
+.. |cdap-apps| replace:: **CDAP Applications:**
+.. _cdap-apps: cdap-apps/index.html
 
-- |included-applications|_
+- |cdap-apps|_
 
   - **Big Data without Big Development:** How to use CDAP "out-of-the-box" to solve problems and use cases 
-  - **Introduction to Included Applications:** Applications that are reusable through configuration and
+  - **Introduction to CDAP Applications:** Applications that are reusable through configuration and
```

Comment:
This seems confusing, as a reader could think this is actually "CDAP apps" in general, which they can write themselves.

How about "pre-built apps" or "default apps"? Also, would "artifacts" make sense instead of "apps"?